### PR TITLE
Add ST-AEDS ASR recipe

### PR DESCRIPTION
**Pull request by JacobPercy**
## What did you change?

Add ST-AEDS (OpenSLR SLR45) ASR recipe.

- `egs2/st_aeds/asr1`
- `egs2/README.md`
- `egs2/TEMPLATE/asr1/db.sh`

---

## Why did you make this change?

Add a reproducible ST-AEDS ASR baseline.

---

## Is your PR small enough?

Yes. 25 files changed.

---

## Additional Context

Baseline: char RNN CTC/attention, no LM.

| split | utts |
|---|---:|
| train | 3038 |
| dev | 400 |
| test | 400 |

| set | WER | CER |
|---|---:|---:|
| dev | 54.4 | 22.4 |
| test | 54.1 | 21.6 |

### CODE REVIEW

**Review by sw005320 — COMMENTED** [review]
Thanks for making a PR.
Several comments

- Can you add an HF link of your model to README.md?
- Can you check whether the text information is overlapped or not? I'm concerned that the same sentences can be used for both training adn test
- Why not using transformer or its variants instead of RNN?
- With such a small scale training data, fine-tuning is more practical. You can refer to some of the fine-tuning recipes and apply them to this data.

**Comment by JacobPercy** [comment]
I've just updated to respond to the review.

Main changes:
- Replaced the RNN baseline with Whisper Medium full fine-tuning.
- Added the Hugging Face model link to `egs2/st_aeds/asr1/README.md`.
- Added normalized transcript-overlap checking across train/dev/test.
- Updated the recipe README with final WER/CER results.

Hugging Face model:
https://huggingface.co/JacobPercy/st_aeds_asr_whisper_medium_finetune

No normalized transcript overlap was found across train/dev/test.

Final results are using beam10 decoding, no CTC, no LM:

| split | WER | CER |
|---|---:|---:|
| dev | 3.9 | 2.7 |
| test | 2.1 | 0.9 |

Thanks
